src/routes/microscope/operations/microscope-dlq-operations.ts

```diff
@@ -0,0 +1,88 @@
+import { Request, Response } from "express";
+import Logger from "bunyan";
+import { getLogger } from "config/logger";
+import { optionalRequire } from "optional-require";
+
+const { DlqService } = optionalRequire("@atlassian/sqs-queue-dlq-service", true) || {};
+
+const log: Logger = getLogger("microscope-dlq");
+
+type RequeueMessageRequest = {
+	messageId: string;
+	receiptHandle: string;
+	body: string;
+	messageAttributes: object;
```

Comment:
Do we know what properties we're expecting for this? Better to pass another type to this if we do.

Reply:
We don't - it's whatever the AWS API client call returns as per https://bitbucket.org/atlassian/dlq-manager/src/master/